.contrib/Parser/DATAS/02 - Outdoor Zones/13 Shadowlands/Revendreth/Treasure Chests.lua

```diff
@@ -417,13 +437,20 @@ _.Zones =
 					["g"] = {
 						i(180589),	-- Soullocked Sinstone (PET!)
 						i(180395),	-- Nathrezim Crusader's Hauberk
+						i(180400),	-- Inquisitions Final Judgment
+						i(180339),	-- Shadow-Lined Chalice
 					},
 				}),
 				o(351540, {	-- Secret Treasure
-					["description"] = "Find a |cFFFFFFFFDredger's Hammer|r in the surrounding area (they sparkle), and use it to open the |cFFFFFFFFJammed Door|r in front of the treasure.\n\nIf the door closes behind you, use the |cFFFFFFFFEscape Shovel|r on the wall.",
+					["description"] = "Find a |cFFFFFFFFDredger's Hammer|r in the surrounding area (they sparkle), and use it to open the |cFFFFFFFFJammed Door|r in front of the treasure.\nRequires |cFFFFD700It Used to Be Quiet Here|r completed in order to see hammers.\n\nIf the door closes behind you, use the |cFFFFFFFFEscape Shovel|r on the wall.",
```

Comment:
If some 'Thing' requires a Quest to be completed for the Thing to become available, we can add a `sourceQuests` tag to indicate this in game as well.
Ex. `["sourceQuests"] = { ### }`